test/tests/vectorpostprocessors/parallel_consistency/parallel_consistency.i

```diff
@@ -0,0 +1,79 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+
+  # To make this deterministic
```

Comment:
Partial comment?

Reply:
No.. ?

That block is there "to make this deterministic"